packages/plugin-core/src/views/ShowMeHowView.ts

```diff
@@ -0,0 +1,39 @@
+import vscode from "vscode";
+
+const getShowMeHowViewHtml = ({ name, src }: { name: string; src: string }) => {
+  return `<!DOCTYPE html>
+  <html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
can this expand to take the full width of the web view?

https://www.loom.com/i/b508dd004d054f68b6754301bf8e87e1

Reply:
I did try this before, but the gif gets really blurry when expanded to the full width. An alternative is to add a higher resolution gif, but it is around 3MB.
Meanwhile, I did center align the gif with the web view and embedded it inside an anchor which links to the loom video.